src/main/java/frc/trigon/robot/subsystem/pitcher/Pitcher.java

```diff
@@ -0,0 +1,32 @@
+package frc.trigon.robot.subsystem.pitcher;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+public class Pitcher extends SubsystemBase {
+    private final static Pitcher INSTANCE = new Pitcher();
+    private final WPI_TalonSRX motor = PitcherConstants.MOTOR;
+
+    public static Pitcher getInstance() {
+        return INSTANCE;
+    }
+
+    public void setTargetAngle(double degrees) {
+        double ticks = Conversions.degreesToMagTicks(degrees);
```

Comment:
local variable is unneeded

Reply:
i fix that i working in that and i didnt do commit yet

Reply:
then when you commit resolve the conversation